code/game/machinery/dance_machine.dm

```diff
@@ -46,24 +53,43 @@
 
 /obj/machinery/jukebox/Initialize(mapload)
 	. = ..()
-	var/list/tracks = flist("[global.config.directory]/jukebox_music/sounds/")
-
-	for(var/S in tracks)
-		var/datum/track/T = new()
-		T.song_path = file("[global.config.directory]/jukebox_music/sounds/[S]")
-		var/list/L = splittext(S,"+")
-		if(L.len != 3)
-			continue
-		T.song_name = L[1]
-		T.song_length = text2num(L[2])
-		T.song_beat = text2num(L[3])
-		songs |= T
-
-	if(songs.len)
+	songs = load_songs_from_config()
+	if(length(songs))
 		selection = pick(songs)
 
+/// Loads the config sounds once, and returns a copy of them.
+/obj/machinery/jukebox/proc/load_songs_from_config()
+	var/static/list/config_songs
+	if(isnull(config_songs))
+		config_songs = list()
+		var/list/tracks = flist("[global.config.directory]/jukebox_music/sounds/")
+		for(var/track_file in tracks)
+			var/datum/track/new_track = new()
+			new_track.song_path = file("[global.config.directory]/jukebox_music/sounds/[track_file]")
+			var/list/track_data = splittext(track_file, "+")
+			if(length(track_data) != 3)
+				continue
+			new_track.song_name = track_data[1]
+			new_track.song_length = text2num(track_data[2])
+			new_track.song_beat = text2num(track_data[3])
+			config_songs += new_track
+
+		if(!length(config_songs))
+			// Includes title3 as a default for testing / "no config" support, also because it's a banger
+			var/datum/track/default_track = new()
+			default_track.song_path = 'sound/ambience/title3.ogg'
+			default_track.song_name = "Tintin on the Moon"
+			default_track.song_length = 3 MINUTES + 52 SECONDS
+			default_track.song_beat = 1 SECONDS
+			config_songs += default_track
```

Comment:
couldn't we just have a subtype of `/datum/track` with all this baked in as our default song so it can be re-used elsewhere if need be rather than edit the full datum? could also be useful for `initial()` memes _just_ in case. optional but i think it's a good idea.